package.json

```diff
@@ -29,20 +29,25 @@
     "lint:php": "composer lint",
     "format": "npm-run-all --parallel format:*",
     "format:php": "composer format",
+    "format:js": "node_modules/.bin/prettier --write js/src/*.js",
     "test": "npm-run-all --parallel test:js test:php",
     "test-with-coverage": "npm-run-all --parallel test:js:coverage test:php:coverage",
     "test:js": "wp-scripts test-unit-js js/src",
     "test:js:coverage": "wp-scripts test-unit-js js/src --coverage --coverageDirectory=tests/coverage/js",
     "test:php": "composer test",
     "test:php:coverage": "composer test-coverage",
     "docker": "docker-compose run --rm --workdir=/var/www/html/wp-content/plugins/foo-bar wordpress",
-    "readme": "composer readme"
+    "readme": "composer readme",
+    "xdebug:status": "bin/xdebug status",
+    "xdebug:start": "bin/xdebug start",
+    "xdebug:stop": "bin/xdebug stop"
   },
   "devDependencies": {
     "@wordpress/block-editor": "3.6.0",
     "@wordpress/eslint-plugin": "3.4.1",
     "@wordpress/scripts": "7.0.1",
-    "npm-run-all": "4.1.5"
+    "npm-run-all": "4.1.5",
+    "wp-prettier": "git+https://github.com/Automattic/wp-prettier.git#wp-prettier-1.18.2"
```

Comment:
I don't think we should add this to the repo when `wp-scripts` comes with the `format-js` command.

Reply:
See my comment above. Thought on this one I might be missing something as it is new architecture for me.